.travis.yml

```diff
@@ -2,68 +2,46 @@ sudo: false
 language: php
 
 php:
-  - 7.0.22
+  - 7.0
   - 7.2
+  
+env:
+  - WORDHAT_DRIVER=wpphp
+  - WORDHAT_DRIVER=wpcli
+  
+services:
+  - docker
+
+#before_cache:
+#  # Save tagged docker images
+#  - mkdir -p $HOME/docker && docker images -a --filter='dangling=false' --format '{{.Repository}}:{{.Tag}} {{.ID}}' | xargs -n 2 -t sh -c 'test -e $HOME/docker/$1.tar.gz || docker save $0 | gzip -2 > $HOME/docker/$1.tar.gz'
```

Comment:
Is this `$HOME/docker` stuff going to be used? Does it work now?

Reply:
Those commented out things are for a future optimization at the moment travis is pulling in all the images from scratch each time, which is very safe, but not very quick. That $HOME/docker will be a place to put and restore images from travis cache.

Reply:
nice plan!